lib/generators/templates/atmosphere.rb

```diff
@@ -0,0 +1,112 @@
+Atmosphere.setup do |config|
```

Comment:
Unused block argument - `config`. You can omit the argument if you don't care about it.